test/source/tests/compose.ts

```diff
@@ -66,6 +68,27 @@ export const defineComposeTests = (testVariant: TestVariant, testWithBrowser: Te
       })
     );
 
+    test(
+      'compose - strip emojis in a recipient email address',
+      testWithBrowser(async (t, browser) => {
+        const acct = 'flowcrypt.compatibility@gmail.com';
+        await BrowserRecipe.setupCommonAcctWithAttester(t, browser, 'compatibility', {
+          google: { acctAliases: flowcryptCompatibilityAliasList },
+        });
+        const recipientEmail = 'User ⭐ Name <test@email.com>';
+        const msgPwd = 'super hard password for the message';
+        const subject = 'Test Sending Message With Recipient Name Contains Emoji';
+        const composePage = await ComposePageRecipe.openStandalone(t, browser, 'compatibility');
+        await ComposePageRecipe.selectFromOption(composePage, acct);
+        await ComposePageRecipe.fillMsg(composePage, { to: recipientEmail }, subject);
+        await ComposePageRecipe.sendAndClose(composePage, { password: msgPwd });
+        const sentMsg = (await GoogleData.withInitializedData(acct)).searchMessagesBySubject(subject)[0];
+        const rawMessage = await Parse.convertBase64ToMimeMsg(sentMsg.raw!);
+        const toHeader = rawMessage.headers.get('to') as AddressObject;
+        expect(!toHeader.text.includes('⭐')).to.equal(true);
```

Comment:
let's make it simpler:
```ts
expect(toHeader.text).to.not.include('⭐');
```

Reply:
Will do - thank you. In addition to this, I have to write the test in the mock endpoint for shared-tenant-fes `/shared-tenant-fes/api/v1/message` since now it will be in the mock endpoint's request body. thanks again.